tests/test_unifi_client.py

```
@@ -0,0 +1,343 @@
+                "cpu_usage": 25.5,
+                "mem_usage": 60.2,
+                "temperature": 45.3,
+                "loadavg_1": 1.2,
+                "loadavg_5": 1.0,
+                "loadavg_15": 0.8,
+            }
+        ]
+    }
+
+
+def test_site_path_construction(unifi_client):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_site_path_construction`

Add return type annotation: `None`

(ANN201)